app/helpers/invite_helper.rb

```diff
@@ -0,0 +1,19 @@
+module InviteHelper
```

Comment:
this is a module, it shouldn't be called helper

Reply:
Ok @Ana06 updating it.

Reply:
@Ana06 created a concern, is it ok now?